src/lighteval/tasks/multilingual/tasks.py

```diff
@@ -520,18 +531,500 @@
+        prompt_function=get_qa_prompt_function(
+            Language.SWAHILI,
+            lambda line: {
+                "question": line["question"],
+                "context": line["context"],
+                "choices": [line["answer"]],
+            },
+        ),
+        suite=("custom",),
+        hf_repo="HuggingFaceFW-Dev/KenSwQuAD",
+        hf_subset="default",
+        evaluation_splits=("test",),
+        few_shots_split="validation",
+        metric=(
+            multilingual_quasi_exact_match_metric(Language.SWAHILI, "prefix"),
+            multilingual_quasi_f1_score_metric(Language.SWAHILI),
+        ),
+        generation_size=400,
+        stop_sequence=("\n",),
+    )
+]
+
+chinese_squad_tasks = [
+    LightevalTaskConfig(
+        name=f"chinese_squad_{Language.CHINESE.value}",
+        prompt_function=get_qa_prompt_function(
+            Language.CHINESE,
+            lambda line: {
+                "question": line["question"],
+                "context": line["context"],
+                "choices": [ans for ans in line["answers"]["text"] if len(ans) > 0],
+            },
+        ),
+        suite=("custom",),
+        hf_repo="HuggingFaceFW-Dev/ChineseSquad",
+        hf_subset="default",
+        evaluation_splits=("validation",),
+        few_shots_split="train",
+        metric=(
+            multilingual_quasi_exact_match_metric(Language.CHINESE, "prefix"),
+            multilingual_quasi_f1_score_metric(Language.CHINESE),
+        ),
+        generation_size=400,
+        stop_sequence=("\n",),
+    )
+]
+
+cmrc2018_tasks = [
+    LightevalTaskConfig(
+        name=f"cmrc2018_{Language.CHINESE.value}",
+        prompt_function=get_qa_prompt_function(
+            Language.CHINESE,
+            lambda line: {
+                "question": line["question"],
+                "context": line["context"],
+                "choices": [ans for ans in line["answers"]["text"] if len(ans) > 0],
+            },
+        ),
+        suite=("custom",),
+        hf_repo="clue/clue",
+        hf_subset="cmrc2018",
+        evaluation_splits=("trial",),
+        few_shots_split="train",
+        generation_size=400,
+        metric=(
+            multilingual_quasi_exact_match_metric(Language.CHINESE, "prefix"),
+            multilingual_quasi_f1_score_metric(Language.CHINESE),
+        ),
+        stop_sequence=("\n",),
+    )
+]
+
+indicqa_tasks = [
+    LightevalTaskConfig(
+        name=f"indicqa_{language.value}",
+        prompt_function=get_qa_prompt_function(
+            language,
+            lambda line: {
+                "question": line["question"],
+                "context": line["context"],
+                "choices": [ans for ans in line["answers"]["text"] if len(ans) > 0],
+            },
+        ),
+        suite=("custom",),
+        hf_repo="ai4bharat/IndicQA",
+        hf_subset=f"indicqa.{LangCodeLanguage.get(language.value).language}",
+        hf_revision="92d96092ae229950973dac3b9998f8b3a8949b0a",
```

Comment:
why the specific revision?

Reply:
whenever I use trust_dataset I lock the revision to prevent any attack vector

Reply:
Smart, specify it in comments so it does not get changed accidentally later